plugins/modules/azure_rm_notificationhub_info.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2020 Praveen Ghuge (@praveenghuge), Karl Dasan (@karldas30)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+from ansible_collections.azure.azcollection.plugins.module_utils.azure_rm_common import AzureRMModuleBase
+
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: azure_notificationhub_info
+short_description: Get Azure Notification Hub
+description:
+    - Get facts of Azure Notification Hub
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group.
+        required: True
+    name:
+        description:
+            - The name of the Notification hub.
+        required: True
+
+
+extends_documentation_fragment:
+- azure.azcollection.azure
+
+
+author:
+    - Praveen Ghuge (@praveenghuge)
+    - Karl Dasan (@karldas30)
+'''
+
+
+EXAMPLES = '''
+  - name: Get facts of specific notification hub
+    community.azure.azure_rm_notificationhub_info:
+      resource_group: myResourceGroup
+      name: myNotificationHub
+
+'''
+
+RETURN = '''
+'''
+
+try:
+    from msrestazure.azure_exceptions import CloudError
+except ImportError:
+    # This is handled in azure_rm_common
+    pass
+
+
+class AzureNotificationHubInfo(AzureRMModuleBase):
+    def __init__(self):
+
+        self.module_arg_spec = dict(
+            resource_group=dict(
+                type='str',
+                required=True
+            ),
+            namespace_name=dict(
```

Comment:
Argument 'namespace_name' is listed in the argument_spec, but not documented in the module documentation